astropy/modeling/projections.py

```diff
@@ -359,9 +359,9 @@ def _validate_mu(mu):
             raise ValueError("Zenithal perspective projection is not defined for mu=-1")
         return mu
 
-    mu = Parameter(default=0.0, setter=_validate_mu)
-    phi0 = Parameter(default=0.0, getter=_to_orig_unit, setter=_to_radian)
-    theta0 = Parameter(default=0.0, getter=_to_orig_unit, setter=_to_radian)
+    mu = Parameter(default=0.0, setter=_validate_mu, description="Distance from point of projection to center of sphere, (Default=0)")
```

Comment:
I don't think we should give defaults here. If you print the parameter, you will see the value. So, I suggest to consistently remove all defaults from all descriptions.

Reply:
Or `print(Model.Parameter.default)`...